08_cell_line_prediction/plot_stratified_drug_response.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>Do you know what the baseline AUPR is for these? I know you said the classes are imbalanced so is it possible that a mean AUPR of ~.3 is actually better (or worse) than the baseline?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I assume by "baseline AUPR" you mean something like <a href="https://stats.stackexchange.com/a/266989" target="_blank">what's described here</a>? That would definitely be useful to look at. My guess is that our "shuffled" models here are actually outperforming that baseline in most cases - since we're shuffling labels within each cancer type (i.e. maintaining label proportions within cancer types), I would imagine that just predicting a different mean value for each cancer type would outperform the overall label proportion across the dataset.</p><p> </p><p>I added this to my "outstanding issues" <a href="https://github.com/greenelab/pancancer-evaluation/issues/53" target="_blank">GitHub issue</a> and I'll try to add it soon. My scripts don't currently save information about the label proportion of the training set after preprocessing/filtering etc. but they probably should.</p>